examples/example-clientlib-ktx-app/app/src/main/java/com/solanamobile/ktxclientsample/activity/MainActivity.kt

```diff
@@ -14,23 +13,21 @@ import com.solanamobile.ktxclientsample.ui.theme.KtxClientSampleTheme
 import dagger.hilt.android.AndroidEntryPoint
 
 @AndroidEntryPoint
-class MainActivity : ComponentActivity(), ActivityResultSender {
+class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        val sender = ActivityResultSender(this)
```

Comment:
i like this, easier for the consumer

Reply:
Yeah this is the _only_ change required for consumers as of this PR.

Reply:
I just want to make sure wee aren't backing into a corner with this. Just want to get steven's perspective so we don't overlook some consumer needs.